### ISSUE TITLE
Please add ability to setup tab tree lvl margin 

### ISSUE THREAD

**Issue by OmTatSat** [mbnuqw/sidebery#676]
## Description
for now i am using this spike

	.Tab[data-lvl="1"] > div.lvl-wrapper {
	  right: 7px !important;
	}
        .Tab[data-lvl="2"] > div.lvl-wrapper {
	  right: 13px !important;
	}
	.Tab[data-lvl="3"] > div.lvl-wrapper {
	  right: 19px !important;
	}
        .Tab[data-lvl="4"] > div.lvl-wrapper {
	  right: 25px !important;
	}

	.Tab[data-lvl="5"] > div.lvl-wrapper {
	  right: 31px !important;
	}
        .Tab[data-lvl="6"] > div.lvl-wrapper {
	  right: 37px !important;
	}
	.Tab[data-lvl="7"] > div.lvl-wrapper {
	  right: 43px !important;
	}
        .Tab[data-lvl="8"] > div.lvl-wrapper {
	  right: 49px !important;
	}

But part of tree lvl dots markers are missing in this way. Are there any other possibility to get this in the right way?

**Comment by mbnuqw** [comment]
This will be fixed in v5, for v4 workaround is
```css
#root .Tab {
 --tabs-indent: 7px;
}

#root[data-tabs-tree-lvl-marks] .Tab .lvl-wrapper:before {
  left: calc(var(--tabs-lvl-indicator-bg) / 2);
  transform: translateX(-3px);
  background-color: var(--tabs-lvl-indicator-bg);
  box-shadow: calc(var(--tabs-indent) * -1) 0 0 0 var(--tabs-lvl-indicator-bg),
              calc(var(--tabs-indent) * -2) 0 0 0 var(--tabs-lvl-indicator-bg),
              calc(var(--tabs-indent) * -3) 0 0 0 var(--tabs-lvl-indicator-bg),
              calc(var(--tabs-indent) * -4) 0 0 0 var(--tabs-lvl-indicator-bg),
              calc(var(--tabs-indent) * -5) 0 0 0 var(--tabs-lvl-indicator-bg),
              calc(var(--tabs-indent) * -6) 0 0 0 var(--tabs-lvl-indicator-bg),
              calc(var(--tabs-indent) * -7) 0 0 0 var(--tabs-lvl-indicator-bg),
              calc(var(--tabs-indent) * -8) 0 0 0 var(--tabs-lvl-indicator-bg),
              calc(var(--tabs-indent) * -9) 0 0 0 var(--tabs-lvl-indicator-bg);
}
```

**Comment by OmTatSat** [comment]
Thank you

**Comment by mbnuqw** [comment]
Fixed in v4.10.2
